sources/packages/web/src/services/http/dto/Application.dto.ts

```diff
@@ -88,6 +88,7 @@ export interface ApplicationBaseAPIOutDTO {
   assessmentId?: number;
   data: any;
   applicationStatus: ApplicationStatus;
+  applicationOfferingIntensity: OfferingIntensity;
```

Comment:
Please update the same in API DTO.

![image](https://github.com/user-attachments/assets/24c559fa-6422-4b15-8156-221f890d910d)